src/components/button/button.tsx

```diff
@@ -102,6 +119,39 @@ export class Button {
         }
     }
 
+    @Listen('keydown')
+    handleKeyDown(ev: KeyboardEvent) {
+        if (ev.key === 'Enter' && !this.isActive) {
+            this.isActive = true;
+
+            this.host.focus();
```

Comment:
When I last worked on this issue it seemed like it took away focus from the host, but now when I test it in lime-elements on main it doesn't 🤔
Maybe we don't need this line after all

Reply:
Update: yeah, you don't need it :)